notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2932,6 +2932,51 @@ biological computer [Dr. Yuhara explained that Mechagodzilla was installed with
 nuclear weapons [Sara stated that it was the use of the H-bomb the first attracted Godzilla to Japan.]
 
 
+movie: Dreamcatcher (2003)
+==========================
+
+:: Title
+Dreamcatcher
+
+:: Description
+Dreamcatcher is a 2003 American science fiction horror film based on Stephen
+King's 2001 novel of the same name.
+
+Synopsis: Four friends encounter an invasion of parasitic aliens.
+
+:: Date
+2003-03-21
+
+:: References
+https://en.wikipedia.org/wiki/Dreamcatcher_(2003_film)
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+friendship [The story follows four male friends as they increasingly find themselves at the center of an invasion of parasitic aliens while otherwise minding their own business on their annual hunting trip in Maine.]
+telepathic ability [The four friends had all acquired telepathic powers after having saved a mentally disabled boy from bullies back when they were children. Their using of this special power was explored throughout the story.]
+mind reading ability [The four friends had all acquired the power to read minds after having saved a mentally disabled boy from bullies back when they were children. Their using of this special power was explored throughout the story.]
+extraterrestrial being [The large humanoid alien Mr. Gray was trying to contaminate the Boston water supply with parasitic alien larvae.]
+attack from outer space [Maine was the site of an invasion of parasitic aliens.]
+human vs. monster [Horrible alien worm creatures were going are parasitizing people.]
+monster [Horrible alien worm creatures were going are parasitizing people. In the end, the surviving friends faced off against a large alien worm creature.]
```

Comment:
"were going are"?

Reply:
Also sounds like a motivation for "human vs. ..."

Reply:
Whoops. Changed "were going are" to "were going around". Typo...

And we have **human vs. monster** in there, too. So we're covered on the "human vs. ..." front.